### ISSUE TITLE
Add tag for latest nonroot variant

### ISSUE THREAD

**Issue by kraftner** [almarklein/timetagger#411]
While working on #409 I noticed that currently there is no tag pointing to the latest nonroot-variant which means one needs to put a specific version in the `docker-composer.yml` when using it.

No idea how/if this can be done and how hard it would be but it would indeed be a nice thing to have.

**Comment by brtwrst** [comment]
I agree - a "latest" tag for the latest nonroot image would be helpful.
Examples:
- `ghcr.io/almarklein/timetagger:nonroot`
- `ghcr.io/almarklein/timetagger:latest-nonroot`
- `ghcr.io/almarklein/timetagger:nonroot-latest`

**Comment by anantakrishna** [comment]
Also, having tags based on major versions would be perfect. Such as `v23` and `v23-nonroot`.

**Comment by skrenes** [comment]
It appears there was an attempt to address the root permissions issue, which may have faced a setback or regression. Isn't the primary need for root privileges just to bind to port 80? To simplify, the example could default to `TIMETAGGER_BIND=0.0.0.0:3000`, removing the necessity for root privileges. For those who need to bind to port 80, they can modify the docker-compose.yml to set `user: 0` (root), allowing the script to attempt the port binding. This will succeed if the container is running as root.

Maybe I'm missing something, but this seems to be an easier approach than having two different containers.

**Comment by danielhass** [comment]
@skrenes regarding you comment here: https://github.com/almarklein/timetagger/pull/523#issuecomment-2815537694

My addition of the `latest-nonroot` tag happend on Jan 6. As you can see here https://github.com/danielhass/timetagger/blob/5451a079f75f2473d4cbf5ace1f87ea5c78173d5/.github/workflows/dockerimage.yml#L81 we need a release (in form of a git tag) to trigger the creation of a new `latest-nonroot` tag. However, timetagger didn't get a new release in the meantime hence this is why there is currently no `latest-nonroot` tag available.

FYI - @almarklein

**Comment by typhoe** [comment]
Hi,

just tried to install timetagger `nonroot` flavor following the [docker-compose.nonroot.yml](https://github.com/almarklein/timetagger/blob/main/deploy/docker-compose.nonroot.yml) template and as it points to the (still) missing tag `latest-nonroot`, it was a little bit confusing...
It's been 6 month since the [last release](https://github.com/almarklein/timetagger/pkgs/container/timetagger/versions?filters%5Bversion_type%5D=tagged), maybe at least add a comment in the compose file ?
Anyway, thank you for your work, and time to test it for me now ;-)

**Comment by danielhass** [comment]
Hi @almarklein, would you be willing to create a new release to address this? Otherwise I can contribute a comment to the compose file or remove the `latest-nonroot` from there until it is really available.
